>From 81a3915aa040f3f6a9684d67f754a0f7f4867987 Mon Sep 17 00:00:00 2001
From: Samuel Holland <samuel@sholland.org>
Date: Sun, 8 Jan 2017 19:14:15 -0600
Subject: [PATCH 2/4] Add missing header for basename()

While it may be transitively included on some systems, libgen.h should
be included explicitly for portability.

Signed-off-by: Samuel Holland <samuel@sholland.org>
---
 tools/profiler/lul/LulElf.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tools/profiler/lul/LulElf.cpp b/tools/profiler/lul/LulElf.cpp
index 6f90d5f13..7a2dd98e7 100644
--- a/tools/profiler/lul/LulElf.cpp
+++ b/tools/profiler/lul/LulElf.cpp
@@ -47,6 +47,7 @@
 
 #include <errno.h>
 #include <fcntl.h>
+#include <libgen.h>
 #include <stdio.h>
 #include <string.h>
 #include <sys/mman.h>
-- 
2.11.0

